examples/mqtt-paho/Makefile

```diff
@@ -0,0 +1,42 @@
+APPLICATION = mqtt-paho-example
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 0
+DEBUG ?= 0
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+#define to use MQTT Paho as a task using MQTTStartTask()
+CFLAGS += -DMQTT_TASK=1
```

Comment:
This CLAFGS is already defined in one of the package Makefile. So it doesn't seem to be needed in the application Makefile.

Reply:
yes it does, because the flags is used by the header to define MQTTStartTask() function

Reply:
I can patch it and remove it though

Reply:
I meant this macro is already added to CFLAGS [here](https://github.com/RIOT-OS/RIOT/pull/12647/files#diff-21e32081d5e896dddff20c9ae4e216baR5), e.g. in the package code, so there's no need to set it again in the application Makefile, where it's not used.